src/locale.en.js

```diff
@@ -30,14 +30,19 @@ export const campsite = {
         description_placeholder: 'Add Description',
         directions: 'Directions',
         directions_placeholder: 'Add descriptive directions',
+        optional: '(Optional)',
+        alternate_sites: 'Alternate Sites',
+        alternate_sites_placeholder: 'Any good backup sites nearby?',
         nearest_town: 'Nearest town',
         nearest_town_placeholder: 'Enter nearest town',
         here_now: 'I am here now',
         accessibility: 'Accessibility',
         facilities: 'Facilities',
         price: 'Price',
         features: 'Features',
+        cell_service: 'Cell Service',
         price_options: {
+            blank: '',
```

Comment:
I added this `blank` option to all pickers, so that they start off unselected. This forces the user to explicitly choose a value. I did the same logic for the cell service stuff below.